src/test/pybind/test_rbd.py

```diff
@@ -2835,15 +2840,37 @@ def test_aio_mirror_image_create_snapshot(self):
         peer_uuid = self.rbd.mirror_peer_add(ioctx, "cluster", "client")
         self.rbd.mirror_mode_set(ioctx, RBD_MIRROR_MODE_IMAGE)
         self.image.mirror_image_disable(False)
-        self.image.mirror_image_enable(RBD_MIRROR_IMAGE_MODE_SNAPSHOT)
 
+        # this is a list so that the local cb() can modify it
+        info = [None]
```

Comment:
I'd set the list item to some bogus value like 123 to ensure that the callback is called and sets it to None on error as expected.